src/app/pages/send-tokens/send-tokens.tsx

```diff
@@ -129,8 +124,9 @@ function SendTokensFormBase() {
           <Suspense fallback={<></>}>
             <SendFormInner
               assetError={assetError}
-              feeEstimations={feeEstimations.estimates}
               nonce={nonce}
+              setAssetIdForValidation={setAssetIdForValidation}
```

Comment:
Per above comment, it'll make much more sense here when reading:

_when the user selects asset, set the asset id for validation_